packages/gatsby-design-tokens/package.json

```diff
@@ -17,7 +17,7 @@
     "./dist/**/*"
   ],
   "scripts": {
-    "build": "microbundle build --sourcemap=false -f es,cjs && agadoo dist/index.esm.js",
+    "build": "(microbundle build --sourcemap=false -f es,cjs && agadoo dist/index.esm.js) || node -e \"console.log('\\nMicrobundle failed, make sure you\\'re using node version 10.16 or higher\\n')\"",
```

Comment:
actually, we should still `exit 1` there

Node <10.16 might not be only reason this build fail, and we might end up publishing broken release

Reply:
😂 Sorry for adding the label!

Reply:
oh good catch

Reply:
![image](https://user-images.githubusercontent.com/1120926/65766398-2610f680-e12b-11e9-8a90-4f8e9453b00b.png)